draft-ietf-jsonpath-base.md

```diff
@@ -1301,7 +1301,7 @@ the comparison is between values which are both numbers or both strings and whic
       the remainder of the first string compares less than the remainder of the second string.
 
 Note that comparisons using the operator `<` yield false if either value being
-compared is an object, array, boolean, or `null`.
+compared is an object, array, boolean, `null`, or (see below) `Nothing`.
```

Comment:
I've cherry-picked this into https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base/pull/454 and tried to improve it there.